environment.yml

```diff
@@ -128,5 +127,6 @@ dependencies:
   # build the interactive terminal
   - jupyterlab >=3.4,<4
   - pip:
+      - flake8==6.0.0
```

Comment:
ah, looks like this will be on conda, the PR just needs merging: https://github.com/conda-forge/flake8-feedstock/pull/60

putting this on hold til then